server/channels/app/limits.go

```diff
@@ -34,3 +41,16 @@ func (a *App) shouldShowUserLimits() bool {
 
 	return a.License() == nil
 }
+
+func (a *App) isHardUserLimitExceeded() (bool, *model.AppError) {
+	if should := a.shouldShowUserLimits(); !should {
+		return false, nil
+	}
```

Comment:
nit: seems like `GetUserLimits` below is already checking for `shouldShowUserLimits` so it wouldn't be needed here?